### PR TITLE
Plus_one Cpp solution Added by Akansha77

### LINKED ISSUE #141: Plus One
You are given a large integer represented as an integer array digits, where each digits[i] is the ith digit of the integer. The digits are ordered from most significant to least significant in left-to-right order. The large integer does not contain any leading 0's.

Increment the large integer by one and return the resulting array of digits.

 

Example 1:

Input: digits = [1,2,3]
Output: [1,2,4]
Explanation: The array represents the integer 123.
Incrementing by one gives 123 + 1 = 124.
Thus, the result should be [1,2,4].
Example 2:

Input: digits = [4,3,2,1]
Output: [4,3,2,2]
Explanation: The array represents the integer 4321.
Incrementing by one gives 4321 + 1 = 4322.
Thus, the result should be [4,3,2,2].
Example 3:

Input: digits = [9]
Output: [1,0]
Explanation: The array represents the integer 9.
Incrementing by one gives 9 + 1 = 10.
Thus, the result should be [1,0].

### PR DESCRIPTION
**Pull request by Akansha77**
Cpp code added by Akansha77 for Plus_one
Closes #141

### CODE REVIEW

**Comment by Akansha77** [comment]
Search In a 2d matrix also done by Akansha77 @bh-g

File: Others/Search_in_a_2D_matrix/question.md
Comment (bh-g) [inline Others/Search_in_a_2D_matrix/question.md:1]: Same as day5 q1

Comment (Akansha77) [reply Others/Search_in_a_2D_matrix/question.md:1]: dii changes done @bh-g

**Comment by Akansha77** [comment]
Search Insert position done @bh-g .

File: Others/Search_in_a_2D_matrix/question.md
Comment (Akansha77) [reply Others/Search_in_a_2D_matrix/question.md:1]: dii is this ok that i am adding all the solutions in this pr? @bh-g

Comment (bh-g) [reply Others/Search_in_a_2D_matrix/question.md:1]: 4 PRs would be better!

**Comment by bh-g** [comment]
Issue #141 closed with #170

File: Others/Search_in_a_2D_matrix/question.md
Comment (Akansha77) [reply Others/Search_in_a_2D_matrix/question.md:1]: dii 4 Pr banani padegi kyaa?

Comment (Akansha77) [reply Others/Search_in_a_2D_matrix/question.md:1]: dii pls tell new @bh-g

**Pull request closed without merging**